compose/xui-frontend.yml

```diff
@@ -0,0 +1,63 @@
+---
+version: '2.4'
+
+services:
+  xui-manage-cases:
+    image: "${XUI_MANAGE_CASES_USE_LOCAL-hmctspublic.azurecr.io/}xui/webapp:${XUI_MANAGE_CASES_TAG:-latest}"
+    container_name: xui-manage-cases
+    environment:
+      FEATURE_APP_INSIGHTS_ENABLED: "true"
+      FEATURE_SECURE_COOKIE_ENABLED: "false"
+      FEATURE_PROXY_ENABLED: "false"
+      FEATURE_TERMS_AND_CONDITIONS_ENABLED: "false"
+      FEATURE_HELMET_ENABLED: "true"
+      FEATURE_OIDC_ENABLED: "false"
+      FEATURE_REDIS_ENABLED: "false"
+
+      MICROSERVICE: xui_webapp
+      PROTOCOL: http
+      XUI_ENV: local
+      NODE_CONFIG_ENV: development
+
+      SERVICES_CCD_COMPONENT_API: http://localhost:3453
+      SERVICES_CCD_DATA_STORE_API: http://ccd-data-store-api:4452
+      SERVICES_IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      SERVICES_IDAM_CLIENT_ID: xuiwebapp
```

Comment:
Minor, but I think the actual client ID in prod is `xui_webapp` so maybe just for consistency worth using that too everywhere this is currently referenced? Also do we need to add the new client to `DATA_STORE_S2S_AUTHORISED_SERVICES`?

Reply:
Yes, we need to add to DATA_STORE_S2S_AUTHORISED_SERVICES

Reply:
changed xuiwebapp to xui_webapp